kolibri/plugins/coach/assets/src/views/common/QuestionDetailLearnerList.vue

```diff
@@ -2,7 +2,7 @@
 
   <div :style="{ backgroundColor: $themeTokens.surface }">
     <h3 class="header">
-      {{ coachStrings.$tr('learnersLabel') }}
+      {{ common$tr('learnersLabel') }}
```

Comment:
I don't actually remember the thinking behind renaming `coachStrings` to ` common$tr`. Do you recall?

Perhaps for the sake of this PR we should leave the naming as-is just to make it a smaller diff? The when we add new common files for core and other plugins, we can decide on final naming conventions.

Reply:
The reason is that we've rewritten the mixin altogether.

So - now the only thing being mixed in is a method `common$tr()` - which takes a key and returns the proper value from the coachStrings object in the commonCoachStrings file.

All of the common files would then have that mixin - might need to rename the mixed-in method when adding a commonCoreStrings file/mixin - or just use `commonCore$tr()` as the core mixin's method name.